src/svgdevicecontext.cpp

```diff
@@ -858,9 +858,9 @@ void SvgDeviceContext::DrawMusicText(const std::wstring &text, int x, int y, boo
 
     int w, h, gx, gy;
 
-    // include the `xlink:` prefix for backwards compatibility with older SVG viewers.
+    // remove the `xlink:` prefix for backwards compatibility with older SVG viewers.
     std::string hrefAttrib = "href";
-    if (m_includeXlink) {
+    if (!m_removeXlink) {
```

Comment:
FYI, I did it this way to avoid needing to strip characters off the string, and instead prepending to the string by default. This avoids needing to make a new copy of the string, or doing any index-based counting to remove the attribute.